catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/OverrideAttributesSupport.java

```diff
@@ -43,31 +43,29 @@ public static void overrideAttributes(
       Metacard currentMetacard = metacardMap.get(contentItem.getId());
       Metacard overrideMetacard = contentItem.getMetacard();
 
-      Metacard updatedMetacard = overrideMetacard(currentMetacard, overrideMetacard, false, false);
+      Metacard updatedMetacard = overrideMetacard(currentMetacard, overrideMetacard, false);
 
       metacardMap.put(contentItem.getId(), updatedMetacard);
     }
   }
 
   public static Metacard overrideMetacard(
-      Metacard currentMetacard,
-      Metacard overrideMetacard,
-      boolean ignoreType,
-      boolean onlyFillNull) {
-    MetacardType updatedMetacardType = currentMetacard.getMetacardType();
-    if (!ignoreType && !MetacardImpl.BASIC_METACARD.equals(overrideMetacard.getMetacardType())) {
-      updatedMetacardType = overrideMetacard.getMetacardType();
-    }
+      Metacard currentMetacard, Metacard overrideMetacard, boolean onlyFillNull) {
 
-    Metacard updatedMetacard = new MetacardImpl(currentMetacard, updatedMetacardType);
+    MetacardImpl updatedMetacard = new MetacardImpl(currentMetacard);
+    updatedMetacard.setType(
+        new MetacardTypeImpl(
+            currentMetacard.getMetacardType().getName(),
```

Comment:
Shouldn't be. The descriptors should be stored in a set so we won't have duplicates. We keep the original (current) metacard type name because that is based on what the input product was determined to be whereas the override doesn't necessarily respect that.